src/gui/TotpExportSettingsDialog.ui

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ui version="4.0">
+ <class>TotpExportSettingsDialog</class>
+ <widget class="QWidget" name="TotpExportSettingsDialog">
+  <property name="geometry">
+   <rect>
+    <x>0</x>
+    <y>0</y>
+    <width>264</width>
+    <height>194</height>
+   </rect>
+  </property>
+  <property name="windowTitle">
+   <string>Scan TOTP Settings</string>
+  </property>
+  <layout class="QVBoxLayout" name="verticalLayout">
+   <item>
+    <widget class="QLabel" name="totpKeyLabel">
+     <property name="font">
+      <font>
+       <pointsize>53</pointsize>
```

Comment:
Does the font really need to be of non-standard size? It looks a bit off compared to the rest of the UI.

Reply:
> Does the font really need to be of non-standard size?

No indeed; this label doesn't render any text though.

Reply:
Generally, do not set font settings inside a UI file. They are absolute and don't scale with the user's DPI configuration.